common/Field_Dimensions.hpp

```diff
@@ -56,6 +57,9 @@ struct Field_Dimensions {
     inline Geometry2d::Rect OurHalf() const { return _OurHalf; }
     inline Geometry2d::Rect TheirHalf() const { return _TheirHalf; }
 
+    //We play on a 2D rectangular field, hence the length "4" of the array
+    inline std::array<Geometry2d::Line, 4> FieldBorders() const { return _FieldBorders; }
```

Comment:
I'll go ahead and remove the inline keyword then. Should I add the & to the other fields in Field_Dimensions as well (i.e. _ourHalf) or are they good as is?